envs/example/defaults.yml

```diff
@@ -40,6 +40,11 @@ memcached:
   user: nobody
   max_connections: 1024
 
+swift:
+  enabled: False
+  haproxy_vip: "{{ fqdn }}"
```

Comment:
Correct... I twas thinking default swift endpoint would be similar to other endpoints. Deploying environment will override swift.haproxy_vip

How should this be handled?